tests/models/test_user.py

```diff
@@ -124,6 +124,34 @@ def test_contributed_property(self):
         projects_contributed_to = project.model.Node.find(Q('contributors', 'eq', self.user._id))
         assert_equal(list(self.user.contributed), list(projects_contributed_to))
 
+    def test_contributor_to_property(self):
+        normal_node = factories.ProjectFactory(creator=self.user)
+        deleted_node = factories.ProjectFactory(creator=self.user, is_deleted=True)
+        bookmark_collection_node = factories.BookmarkCollectionFactory(creator=self.user)
+        collection_node = factories.CollectionFactory(creator=self.user)
+        contributor_to_nodes = [node._id for node in self.user.contributor_to]
```

Comment:
Almost there. Could you ensure that the an invisible contributor will also be returned via contributor_to, since you have the other side of that covered in the next test?